src/views/Viewer.vue

```diff
@@ -707,6 +713,13 @@ export default {
 		onClose() {
 			this.Viewer.onClose()
 		},
+
+		async onDelete() {
+			// let fileInfo = await fileRequest(this.file)
+			let fileInfo = this.fileList[this.currentIndex]
```

Comment:
You can use this.currentFile

Reply:
OK thanks.
How to actually delete this.currentFile from the server?

Reply:
You need to send a delete request to the dav path

Reply:
It's working 👍